lib/elixir/lib/calendar.ex

```diff
@@ -1337,6 +1376,12 @@ defmodule NaiveDateTime do
 
   ## Helpers
 
+  defp discard_timezone_info(naive_datetime) do
```

Comment:
You are completely correct. This indeed is a function that is not used anymore in the current version. I'll remove it.

Glad you noticed it! :smile: